server/server.go

```diff
@@ -15,34 +15,22 @@ import (
 
 	"github.com/elastic/apm-server/processor"
 	"github.com/elastic/beats/libbeat/beat"
-	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
 var debugHandler = flag.Bool("debugHandler", false, "Enable for having catchall endpoint enabled")
 
-type Server struct {
-	config Config
-	http   *http.Server
+type ApmServer struct {
```

Comment:
If we call this just `Apm` we will have `server.Apm` which would explain it better. Agree that `server.Server` is not nice.